votor/src/event_handler.rs

```diff
@@ -124,10 +128,26 @@ impl EventHandler {
 
         // Wait until migration has completed
         info!("{}: Event loop initialized", local_context.my_pubkey);
-        let Some(genesis_block) = migration_status.wait_for_migration_or_exit(&exit) else {
-            // Exited during migration
-            return Ok(());
-        };
+        let mut last_genesis_vote_refresh_time = Instant::now();
```

Comment:
So the earliest we send out genesis vote is after 400ms?

Reply:
yeah we have to freeze at least 2 blocks along with all the normal startup things before we can identify a genesis block